members/H001076.yaml

```diff
@@ -4,6 +4,10 @@ contact_form:
   action: ""
   steps:
     - visit: "https://www.hassan.senate.gov/contact/email"
+    - find:
```

Comment:
do you need the find in addition to the click_on? can click_on be enough?

Reply:
I think `find` is still needed for two reasons:
- to make sure the right element is clicked after the modal shows up
- to add a little bit of a wait time in there in case the newsletter modal takes a little bit to show up. IIRC, `find` has an inherent 5 second wait time until it gives up trying to find a specific element. I think this is more succinct than adding a `wait` step

Reply:
ok